girder/utility/install.py

```diff
@@ -277,6 +290,10 @@ def main():
     web.add_argument('--all-plugins', action='store_true',
                      help='build all available plugins rather than just enabled ones')
     web.add_argument('--plugins', default='', help='comma-separated list of plugins to build')
+    web.add_argument('--watch', action='store_true',
+                     help='watch for changes and rebuild girder core library in dev mode')
```

Comment:
It means that the web client build process is built in dev mode instead of prod, since `watch` is an operation that only really happens in a dev environment.

* dev mode: source maps are built, no minification postprocessing
* prod mode: no source maps, includes minification step